deploy/clowdapp.yaml

```diff
@@ -893,10 +893,10 @@ parameters:
     value: pulp-stage-default-domain-s3
   - name: PULP_AUTHENTICATION_BACKENDS
     description: The AUTHENTICATION_BACKENDS to be used by Pulp.
-    value: "@merge django.contrib.auth.backends.RemoteUserBackend" 
+    value: "django.contrib.auth.backends.RemoteUserBackend,django.contrib.auth.backends.ModelBackend,pulpcore.backends.ObjectRolePermissionBackend"
```

Comment:
This needs to be inside [ ] to be a list. Pulp API fails to start with this string in the settings.

Reply:
Done